packages/terra-functional-testing/lib/test-runner/test-runner.js

```diff
@@ -0,0 +1,48 @@
+const fs = require('fs');
+const path = require('path');
+const Launcher = require('@wdio/cli').default;
```

Comment:
Should we wrap the wdio CLI or should we use the wdio launcher directly?

Reply:
Are you doing this to expose the CLI commands without redefining them?

Reply:
The default export is just the launcher. Only the launcher is being used here:

https://github.com/webdriverio/webdriverio/blob/master/packages/wdio-cli/src/index.js#L76

Does that answer your question?

Reply:
yes that does